devcontainers-cli/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
+# If @devcontainers/cli is already installed, we can skip
+if command -v devcontainer > /dev/null 2>&1; then
+  echo "🥳 @devcontainers/cli is already installed"
```

Comment:
This'll retain similar output to the final "installed" line.

```suggestion
  echo "🥳 @devcontainers/cli is already installed into $(which devcontainer)!"
```